src/dysh/plot/specplot.py

```diff
@@ -1,4 +1,6 @@
-"""Plot a spectrum using matplotlib"""
+"""
+Plot a spectrum using matplotlib
+"""
```

Comment:
?

Reply:
Ah, I just added line breaks to the file's docstring because I think it looks better

Reply:
@tchamberlin & @vcatlett has this been resolved?